config/passport.js

```diff
@@ -3,7 +3,7 @@ const JwtStrategy = require('passport-jwt').Strategy;
 const { PRIVATE_KEY } = require('../config/apiCredentials');
 
 const cookieExtractor = (request) => {
-	var token = null;
+	let token = null;
```

Comment:
Ah yes, this has highlighted something I accidentally glossed over.

`let` alert! :alarm: :alarm:

I think we can get away without needing a let here and just returning the token's value at each point where we currently set it.

Reply:
I think I've sorted this out.